frontend/src/renderer/components/ImageImporter.svelte

```diff
@@ -0,0 +1,148 @@
+        $processState.thumbnailID = Math.floor(Math.random() * 999999999);
+        let msg = {
+            RequestID: $processState.thumbnailID,
+            RequestType: "Thumbnails",
+            RequestData: {
+                names: filePaths,
+            },
+        };
+        console.log(msg);
+        sendMessage(JSON.stringify(msg));
+        filePaths = [];
+    }
+
+    $: if (filePaths?.length > 0) {
+        console.log("Fetching Thumbnails");
+        getThumbnails();
+    }
+
+    $: if (
+        $processState.imageFilePaths.length >= 6 &&
+        !$processState.completedTabs[0]
+    ) {
+        $processState.completedTabs[0] = true;
+    }
+
+    function handleFilesSelect(e) {
+        console.log("files")
+        if (files.accepted.length < MAX_FILES){
+            files.accepted = [];
+            console.log("accepted")
+            const {acceptedFiles, fileRejections} = e.detail;
+            files.accepted = [...files.accepted, ...acceptedFiles];
+            files.rejected = [...files.rejected, ...fileRejections];
+
+            forEach(files.accepted, (f) => {
+                if (!find(filePaths, {id: f.path, name: f.name})) {
+                    filePaths.push({
+                        id: f.path,
+                        name: f.path
+                    });
+                }
+            });
+            $processState.imageFilePaths = [...filePaths];
+        }
+    }
+</script>
+
+<main>
+    <br>
+    <Dropzone
+            on:drop={handleFilesSelect}
+
+            containerStyles="flex: 1;
+                            display: flex;
+                            flex-direction: column;
+                            align-items: center;
+                            padding: 20px;
+                            border-width: 2px;
+                            border-radius: 2px;
+                            border-color: #eeeeee;
+                            border-style: dashed;
+                            background-color: #1D1C1E;
+                            color: #bdbdbd;
+                            outline: none;
+                            transition: border 0.24s ease-in-out;
+                            background-color: #1D1C1E;
+                            border-radius: 10px;
+                            font-size: 30px;
+                            text-align: center;"
+            disableDefaultStyles
+            containerClasses="custom-dropzone">
+<!--        <button>-->
+<!--            {label}-->
+<!--            <div class="icon">-->
+<!--                <svelte:component this={icon} size="1.5x" />-->
+<!--            </div>-->
+<!--        </button>-->
+        <br>
+        Click or Drag and Drop Files Here
+        <br>
+        <br>
+        <article>
+            <ul>
+                {#if $processState.imageFilePaths?.length > 0}
+                    {#each $processState.imageFilePaths as filePath}
+                        <ImageBubble filename={filePath.name} />
```

Comment:
My guess is this is what is causing the image display bug, could it be 'name' isn't the path anymore (it shouldn't be, but I know it was before)